### PR TITLE
Add empty table response

### PR DESCRIPTION
**Pull request by lynn-kim**
## Notion ticket link

[Add empty columns table response to UsersTable and StoryTranslationsTable](https://www.notion.so/uwblueprintexecs/Add-empty-columns-table-response-to-UsersTable-and-StoryTranslationsTable-eed664e5b725466da3611c54e0a564bb)

## Implementation description

- Added a new empty table respone component, accepts array of filters to be set to null

## Steps to test

1. Go to http://localhost:3000/admin as angela merkel
2. Add some filters until there aren't any rows
3. See that the component shows up
4. Check that it clears the filters!
5. Try other combinations of filters 

## What should reviewers focus on?

- Is it safe to assume that the filters can all be set to null? 
- Does it work ok?

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] For backend changes, I have run the appropriate linters: `docker exec -it planet-read_py-backend_1 /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/admin/EmptyTable.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import { Button, Flex, Heading, Text, Tr, Td } from "@chakra-ui/react";
+
+export type EmptyTableProps = {
+  filters: { (data: string | null): void }[];
+};
+
+const EmptyTable = ({ filters }: EmptyTableProps) => {
+  const clearFilters = () => {
+    filters.forEach((f) => {
```
Comment (gaoxk) [inline frontend/src/components/admin/EmptyTable.tsx:11]: nit: `f->setFilter` this gave me a little heart attack it thought you were directly setting state LOL

File: frontend/src/components/admin/EmptyTable.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      f(null);
+    });
+  };
+  return (
+    <Tr>
+      <Td colspan="100%" background="white !important">
+        <Flex
+          direction="column"
```
Comment (gaoxk) [inline frontend/src/components/admin/EmptyTable.tsx:19]: nit: order the props alphabetically (ie `alignItems` first)

**Review by gaoxk — CHANGES_REQUESTED** [review]
your programming speed is immaculate 😍 

> Is it safe to assume that the filters can all be set to null?

Imo thats fine!

UI nit: when just using the filters as normal, the table flickers alot. Previously it wasnt noticeable as it was just an empty table. Now it flickers between a full table of data and the newly implemented screen. Maybe an `isLoading` state can be introduced and if `isLoading == true`, dont display the `EmptyTable.` What do you think?

**Review by Puepis — APPROVED** [review]
tested and LGTM!

**Review by gaoxk — APPROVED** [review]
noice 😎

**Pull request merged**